notebooks/catvton/ov_catvton_helper.py

```diff
@@ -0,0 +1,209 @@
+import os
+from collections import namedtuple
+from pathlib import Path
+import warnings
+
+from detectron2.export import TracingAdapter
```

Comment:
I do not see anything detectrron2 related in dependencies, from what these imports comes from?